js/.prettierrc

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
I don't think that this file is needed since we have eslint config

Reply:
eslint does not provide formatting at the moment. I think it would be a good idea if everyone had the same formatting rules